code/core/src/states/PlayState.java

```diff
@@ -1,43 +1,44 @@
 package states;
 
 import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
 import com.badlogic.gdx.graphics.GL20;
-import com.badlogic.gdx.graphics.OrthographicCamera;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import com.badlogic.gdx.maps.tiled.TiledMap;
-import com.badlogic.gdx.maps.tiled.TmxMapLoader;
 import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
 import com.badlogic.gdx.math.Vector2;
 import com.badlogic.gdx.math.Vector3;
 import com.badlogic.gdx.physics.box2d.Box2DDebugRenderer;
-import com.ece454.gotl.Game;
+import com.ece454.gotl.GotlGame;
 import com.ece454.gotl.Goose;
 
+import handlers.AssetHandler;
 import handlers.GameStateManager;
 import handlers.WorldManager;
+import screens.LevelCompleteScreen;
 
-import static com.ece454.gotl.Game.PIXEL_PER_METER;
-import static com.ece454.gotl.Game.POSITION_ITERATIONS;
-import static com.ece454.gotl.Game.SCALE;
-import static com.ece454.gotl.Game.TIME_STEP;
-import static com.ece454.gotl.Game.VELOCITY_ITERATIONS;
+import static com.ece454.gotl.GotlGame.PIXEL_PER_METER;
+import static com.ece454.gotl.GotlGame.POSITION_ITERATIONS;
+import static com.ece454.gotl.GotlGame.SCALE;
+import static com.ece454.gotl.GotlGame.TIME_STEP;
+import static com.ece454.gotl.GotlGame.VELOCITY_ITERATIONS;
 
-public class PlayState extends State
-{
-    private static final String MAP_PATH = "map/desert_demo.tmx";
+public class PlayState extends State implements Screen {
     private TiledMap tiledMap;
     private OrthogonalTiledMapRenderer tiledMapRenderer;
     private Goose goose;
     private boolean isPressed = false;
     private Vector2 initialPressPos, finalPressPos;
     private Box2DDebugRenderer box2DDebugRenderer;
+    private AssetHandler assetHandler;
 
     public PlayState(GameStateManager gsm)
     {
         super(gsm);
         initialPressPos = new Vector2();
         finalPressPos = new Vector2();
-        tiledMap = new TmxMapLoader().load(MAP_PATH);
+        assetHandler = gsm.getGame().assetHandler;
+        tiledMap = assetHandler.manager.get(assetHandler.MAP_PATH, TiledMap.class);
```

Comment:
Make AssetHandler static, manager private, use getters/setters instead

Reply:
will make the manager private and use getter/setters